src/preferences/dialog/dlgprefcontrolsdlg.ui

```diff
@@ -489,6 +489,23 @@
        </property>
       </widget>
      </item>
+     <item row="12" column="1" colspan="2">
+      <widget class="QCheckBox" name="checkBoxDoubleWidgetSize">
```

Comment:
I know other ratios are common on smartphone screens, but Mixxx does not run on those. I am not sure about laptops.